Src/AutoFixture.xUnit.net2/AutoDataDiscoverer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Ploeh.AutoFixture.Xunit2
+{
+    internal class AutoDataDiscoverer : DataDiscoverer
```

Comment:
Wouldn't internal be better, given that it works, and it doesn't add any value to the public API?

> On 11 Apr 2015, at 21:22, Mark Seemann <notifications@github.com> wrote:
> 
> In Src/AutoFixture.xUnit.net2/AutoDataDiscoverer.cs:
> 
> > @@ -0,0 +1,22 @@
> > +﻿using System;
> > +using System.Collections.Generic;
> > +using System.Linq;
> > +using System.Text;
> > +using System.Threading.Tasks;
> > +using Xunit.Abstractions;
> > +using Xunit.Sdk;
> > +
> > +namespace Ploeh.AutoFixture.Xunit2
> > +{
> > +    internal class AutoDataDiscoverer : DataDiscoverer
> My concern is that we might be relying on undocumented or accidental behaviour.
> 
> A reasonable concern. Why not make it public?
> 
> —
> Reply to this email directly or view it on GitHub.
>